codalab/server/bundle_manager.py

```diff
@@ -841,11 +840,27 @@ def _get_matched_workers(request_queue, workers):
         :param workers: a list of workers
         :return: a list of matched workers
         """
-        tag_match = re.match('(?:tag=)?(.+)', request_queue)
-        if tag_match is not None:
-            return [worker for worker in workers if worker['tag'] == tag_match.group(1)]
+        prefix = "tag="
```

Comment:
It probably would've been most consistent to let people use `tags=` instead... but do you think that's going to be too disruptive to users' workflows?

Reply:
I don't think that anyone actually explicitly passes in `tag=` anymore...(and at this point, they can't actually, since `=` is non-alphanumeric). So maybe we should just remove all of the logic with `tag=` entirely. https://github.com/codalab/codalab-worksheets/blob/master/codalab/worker_manager/worker_manager.py#L188-L194